go/vt/vtgate/planbuilder/expr.go

```diff
@@ -20,10 +20,9 @@ import (
 	"bytes"
 	"fmt"
 
-	"vitess.io/vitess/go/vt/vterrors"
-
 	"vitess.io/vitess/go/vt/sqlparser"
-	"vitess.io/vitess/go/vt/vtgate/engine"
+	"vitess.io/vitess/go/vt/vterrors"
+	popcode "vitess.io/vitess/go/vt/vtgate/engine/opcode"
```

Comment:
I am not sure why the imported name is `popcode`, seems like `opcode` is not used by another package in this file.

Reply:
good point. Multiple files in `planbuilder` were using a variable named `opcode`. So in those files we renamed the package as `popcode`, and the rest is for consistency. I'm happy to change that if you think we should.

Reply:
Oh that makes sense okay! I just wanted to understand, it's fine to keep it as `popcode` :-)

Reply:
besides, it makes me think of `popcorn`! :)